lib/Conversion/StableHLOToTTIR/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ add_mlir_library(TTMLIRStableHLOToTTIR
   LINK_LIBS PUBLIC
   MLIRIR
   MLIRPass
+  StablehloPasses
```

Comment:
Hey @LPanosTT I forgot to mention yesterday that when you link this library, please check if `TTMLIR_ENABLE_STABLEHLO` is enabled (as it is disabled by default). Please see `lib/CMakeLists.txt` for reference.

Reply:
I think that the files in this directory are only linked in the first place if `TTMLIR_ENABLE_STABLEHLO` is enabled. See `lib/Conversion/CMakeLists.txt`